sockeye/utils.py

```diff
@@ -688,7 +689,8 @@ def metric_value_is_better(new: float, old: float, metric: str) -> bool:
         return new < old
 
 
-def cleanup_params_files(output_folder: str, max_to_keep: int, checkpoint: int, best_checkpoint: int, keep_first: bool):
+def cleanup_params_files(output_folder: str, max_to_keep: int, checkpoint: int, best_checkpoint: int, keep_first: bool,
+                         max_params_files_to_cache: int, cache_metric: str, cache_strategy: str):
```

Comment:
can these arguments be made optional? If no strategy is defined for example we would not go into below if block and call the 'average' logic.

Reply:
These could be made optional.